### ISSUE TITLE
FPWD Request for Challenges with Accessibility Guidelines Conformance and Testing, and Approaches for Mitigating Them

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/transitions#241]
# Document title, URLs, estimated publication date
Title: Challenges with Accessibility Guidelines Conformance and Testing, and Approaches for Mitigating Them
Editors' Draft URI: https://w3c.github.io/wcag/conformance-challenges/
Anticipated Publication URI: https://www.w3.org/TR/2020/WD-guidelines-conformance-challenges-20200430/
Anticipated Publication Date: 30 April 2020

# Abstract
This document explores how testability and page-based conformance verification of the WCAG 2.0 and 2.1 accessibility guidelines are challenging to apply to a broad range of websites and web applications. It also explores approaches for mitigating these challenges, to realize as accessible a site as possible.

The challenges covered broadly fall into four main areas:

1.    Numerous provisions need human involvement to test and verify conformance, which is especially challenging to scale for large websites and for dynamic websites;
1.    Large and dynamic sites may have too many changing permutations to validate effectively;
1.    Third parties frequently add and change content on large and dynamic sites;
1.    Applying a web-based, and page-based conformance model can be challenging to do for non-web Information and Communications Technologies (ICT).

The purpose of this document is to help understand those challenges more holistically, and explore approaches to mitigating those challenges, both so that sites can use these mitigation approaches now, and also so that we can address the challenges more fully in future accessibility guidelines such as WCAG 3.0 (now in early development) where the W3C Working Group Charter expressly anticipates a new conformance model.

# Status
This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current W3C publications and the latest revision of this technical report can be found in the W3C technical reports index at https://www.w3.org/TR/.

This is a First Public Working Draft by the Accessibility Guidelines Working Group. This document explores how testability and page-based conformance verification of the WCAG 2.0 and 2.1 accessibility guidelines are challenging to apply to a broad range of websites and web applications. It also explores approaches for mitigating these challenges, to realize as accessible a site as possible. This draft is published to obtain public review of the issues identified and solutions proposed. After sufficient review, the Working Group plans to publish this document as a Working Group Note to inform other work.

Feedback is welcome on any aspect of this document. The Working Group particularly seeks feedback on the following questions:

1.    Are there additional challenges not described in our document?
1.    Are there additions or corrections to the specific Success Criteria discussed in Appendix A and Appendix B?
1.    Is there a priority order in which challenges should be addressed?
1.    Are there aspects of our document that are incorrect or insufficiently defined?
1.   Does the document overlook something that should be addressed?
1.    What do you believe are the most promising approaches for WCAG 3.0 conformance to address or ameliorate the challenges described in this document?

To comment, file an issue in the W3C WCAG GitHub repository. Please indicate your issue is for this document by using the word "Challenges:" as the first word of your issue's title. Although the proposed Success Criteria in this document reference issues tracking discussion, the Working Group requests that public comments be filed as new issues, one issue per discrete comment. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-agwg-comments@w3.org (comment archive).

This document was published by the Accessibility Guidelines Working Group as a First Public Working Draft.

Comments regarding this document are welcome. Please send them to public-agwg-comments@w3.org (archives).

Publication as a First Public Working Draft does not imply endorsement by the W3C Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

This document was produced by a group operating under the W3C Patent Policy. The group does not expect this document to become a W3C Recommendation. W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains Essential Claim(s) must disclose the information in accordance with section 6 of the W3C Patent Policy.

This document is governed by the 1 March 2019 W3C Process Document. 

# Is it a delta specification intended to become a W3C Recommendation?
no

# Link to group's decision to request transition
https://lists.w3.org/Archives/Public/w3c-wai-gl/2020AprJun/0208.html

# Information about implementations known to the Working Group
Note-track, no implementation

**Comment by plehegar** [comment]
I recommend using "accessibility-conformance-challenges".

Approved.

**Comment by plehegar** [comment]
https://www.w3.org/TR/2020/WD-accessibility-conformance-challenges-20200501/
